common/src/main/java/com/sequenceiq/cloudbreak/quartz/cleanup/service/UMSCleanupJobService.java

```diff
@@ -42,7 +42,10 @@ public void schedule() {
         JobDetail jobDetail = buildJobDetail();
         Trigger trigger = buildJobTrigger(jobDetail);
         try {
+            JobKey jobKey = jobDetail.getKey();
+            LOGGER.info("Unscheduling instance metadata archiver job for stack with key: '{}' and group: '{}'", jobKey.getName(), jobKey.getGroup());
```

Comment:
this is ums cleanup, not instance metadata archiver - also this kind of logs occur in multiple job service, so please fix these

Reply:
Thanks, it was a copy-paste issue, but fixed. Please re-review 😄